front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -359,6 +381,42 @@ export default function LabsTranscriptsIndex({
     return updateIsActive(transcriptConfigurationId, isActive);
   };
 
+  const getDefaultConfiguration = async (provider: string) => {
+    const response = await fetch(
+      `/api/w/${owner.sId}/labs/transcripts/default?provider=${provider}`
+    );
+
+    if (response.ok) {
+      const defaultConfigurationRes = await response.json();
+      const defaultConfiguration: LabsTranscriptsConfigurationResource =
+        defaultConfigurationRes.configuration;
+
+      return defaultConfiguration;
+    }
+
+    return null;
+  };
+
+  const saveApiConnection = async (
```

Comment:
Same this shoul live in the swr folder. Example [here](https://github.com/dust-tt/dust/blob/06ac2f44d07ee6bfc7801eba6becada47b6314e5/front/lib/swr/spaces.ts#L233).

Reply:
This one has no SWR mutation and has no hook behavior at all. It doesn't make sense to put in the SWR folder unless it does